src/pages/projects.tsx

```diff
@@ -1,56 +1,17 @@
-import React from 'react'
 import { graphql } from 'gatsby'
-import { Project } from '../../plugins/gatsby-source-cd-airtable/src/interfaces/project'
-import { Layout, Section, SectionContent } from 'components/layout'
-import { useTranslation } from 'gatsby-plugin-react-i18next'
-
-interface ProjectsPageProps {
-  data: {
-    allProject: {
-      edges: {
-        node: Project
-      }[]
-    }
-  }
-}
-
-const ProjectsPage: React.FC<ProjectsPageProps> = ({
-  data,
-}: ProjectsPageProps) => {
-  const { t } = useTranslation()
-  return (
-    <Layout crumbs={[{ label: t('pages.projects.navigation.projects') }]}>
-      <Section>
-        <SectionContent>
-          <h1>{t('pages.projects.title')}</h1>
-          <p>{t('pages.projects.description')}</p>
-          <ul>
-            {data.allProject.edges.map((project) => (
-              <li key={project.node.originalId}>
-                <ul data-cy="project">
-                  <li data-cy="project__name">{project.node.name}</li>
-                  <li data-cy="project__tagline">{project.node.tagline}</li>
-                </ul>
-              </li>
-            ))}
-          </ul>
-        </SectionContent>
-      </Section>
-    </Layout>
-  )
-}
+export { default } from 'page-components/projects'
 
+// https://github.com/gatsbyjs/gatsby/issues/4984
```

Comment:
@HormCodes  :(

Reply:
Ano, toto je bohužel nutné. Můžeš prosím nahradit link slovním komentářem? 

Např. zde `// Page query needs to be in `pages` directory` a v komponentě `// Data are coming from page query defined in 'pages/project.tsx'`? 🙂

Reply:
ok, muzu, ale stojí to za to? Chci říct, udržovat stránky v jiném adresáři než vyhrazené gatsby `pages`.